roles/transformers/tasks/deb-packages.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
These file names are not very consistent. Eg: yum_repo.yml 
At least for tasks we should be consistent with the naming (either use OS related tasks or package manager related tasks, etc.)

Reply:
The reason I've done this is to avoid code duplication: having one file for yum and one for dnf that are identical